app/controllers/users_controller.rb

```diff
@@ -1,7 +1,57 @@
+# frozen_string_literal: true
+
 class UsersController < Clearance::UsersController
-  private
+  before_action :current_user, only: [:show, :edit, :update]
+  before_action :check_for_cancel, only: [:edit, :update]
+
+  # Could be useful for a future issue
+  # before_action :another_user
+
+  # def another_user
+  #   unless user_url.end_with?(current_user.id.to_s)
+  #     redirect_to request.referer
+  #   end
+  # end
+
+  def check_for_cancel
+    if params[:commit] == "Cancel"
+      redirect_to user_url
+    end
+  end
+
+  def index
+    @users = User.all
+  end
+
+  def show
+  end
+
+  def edit
+  end
 
-  def user_params
-    params.require(:user).permit(:name, :phone, :email, :password)
+  def update
+    respond_to do |format|
+      if @user.update(user_params)
+        format.html { redirect_to user_url }
+      else
+        format.html { render :edit }
+      end
+    end
   end
+
+  def destroy
+    @user = User.find(params[:id])
```

Comment:
Chatted offline

- https://github.com/thoughtbot/clearance#access-control
- Please, try to not make this kind of queries without at least the logged user is allowed to perform this kind of actions

Reply:
This is temporary, we needed to check if the route of the delete button worked, we are going to make an issue to refactor this, or it's necessary to change before merge the branch?